docs/nima/openapi.adoc

```diff
@@ -0,0 +1,26 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
This file is no longer needed. (We used to have just a single openapi file, but not we have multiple in the `/openapi` subdirectory, including the correct version of this file.)